warmup.py

```diff
@@ -103,7 +105,18 @@ def energy(self, r):
             If ``r`` is 0, the energy is :py:obj:`numpy.inf`.
 
         """
-        pass
+        r = np.atleast_1d(r)
```

Comment:
Don't forget that you need to import numpy in this file (after the file's docstring). It should look like:
```python
import numpy as np
```